boards/stm32f746g-disco/Makefile.include

```diff
@@ -0,0 +1,11 @@
+# we use shared STM32 configuration snippets
+INCLUDES += -I$(RIOTBOARD)/common/stm32/include
+
+# this board uses openocd
+PROGRAMMER ?= openocd
+
+# this board has an on-board ST-link adapter
+OPENOCD_DEBUG_ADAPTER ?= stlink
+
+# openocd programmer is supported
+PROGRAMMERS_SUPPORTED += openocd
```

Comment:
Can't you include `stm32.inc.mk`?

Reply:
Including it will set `OPENOCD_RESET_USE_CONNECT_ASSERT_SRST` to 1 and I'm not sure this is actually needed for this board. No other stm32xxx-disco board are including it.

Reply:
But I think all the other programmers are also supported, why not then:

```
OPENOCD_RESET_USE_CONNECT_ASSERT_SRST ?= 0
include stm32.inc.mk
```